### PR TITLE
ch: setup-docker

### PR DESCRIPTION
**Pull request by phanuelnd**
#### What does this PR do?
-implement docker
#### Description of Task to be completed
-implement image of the backend application
-implement database container
-implement shared network (communication) between two containers database and server
#### How should this be manually tested?
-install docker
-clone the repo using git clone repo's URL
-change branch git checkout ch-setup-docker 
-create a .env file and put the required environment variables from .env.example file
-run `docker-compose up --build` 
-run `docker exec ecommerce-bn npx sequelize db:migrate`
- run `docker exec ecommerce-bn npm run db:seeds`
#### What are the relevant pivotal tracker stories?
-Setup Docker for the backend  card #11

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+services:
+  db:
+    container_name: postgres
+    restart: always
+    image: postgres
+
+    environment:
+      - POSTGRES_PASSWORD=${DB_PASSWORD}
+      - POSTGRES_USER=${DB_USERNAME}
+      - POSTGRES_DB=${DB_NAME}
```
Comment (wakaflorien) [inline docker-compose.yml:11]: Getting some credentials issue

**Review by wakaflorien — CHANGES_REQUESTED** [review]
- can we add run script in package.json

**Comment by sostene12** [comment]
> * can we add run script in package.json

let me change it

**Comment by sostene12** [comment]
> > * can we add run script in package.json
> 
> let me change it

done it

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+version: '3.9'
+services:
+  db:
+    container_name: postgres
+    restart: always
+    image: postgres
+
+    environment:
+      - POSTGRES_PASSWORD=${DB_PASSWORD}
+      - POSTGRES_USER=${DB_USERNAME}
```
Comment (wakaflorien) [inline docker-compose.yml:10]: - can we add ports?
- include needed env variables in README.md file @sostene12 @oliviertech212

**Review by wakaflorien — APPROVED** [review]

**Pull request merged**